codice/ddf
# [2.19.x] DDF-UI-178 G-8254 actions UX improvements

#### What does this PR do?
Removes the second "Actions" in the inspector view and pulls the dropdown menu items to the same level as the "Filter". Fixes https://github.com/codice/ddf-ui/issues/178

#### Who is reviewing it? 

@andrewzimmer @hayleynorton @zta6 
#### Select relevant component teams: 

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@bdeining 

#### How should this be tested?

- build/profile install
- execute a search that returns results
- click on a result and open the inspector view
- the inner bar should have "Filter", "Rearrange", and "Show"(or "Hide" if in the Details view (see Screenshots)
- verify that if "Allow Editing" is checked in the admin console, then you see "Add" and "Remove" (see Screenshots)
- verify that if "Allow Editing" is cleared in the admin console, then you no longer see "Add" and "Remove" (see Screenshots)
- verify that all the options function as expected

#### Any background context you want to provide?

#### What are the relevant tickets?
https://github.com/codice/ddf-ui/issues/178

#### Screenshots


### Before
<img width="538" alt="Screen Shot 2020-04-27 at 3 15 12 PM" src="https://user-images.githubusercontent.com/25390369/80421563-e9140a80-8899-11ea-8292-48acd566b3e4.png">


### After
with "Allow Editing" checked: 
<img width="542" alt="Screen Shot 2020-04-27 at 2 59 26 PM" src="https://user-images.githubusercontent.com/25390369/80420468-0c3dba80-8898-11ea-9ef9-7e7268ead532.png">

"Details" tab has "Hide": 
<img width="222" alt="Screen Shot 2020-04-27 at 2 58 26 PM" src="https://user-images.githubusercontent.com/25390369/80420472-0cd65100-8898-11ea-85c5-a1af9faebc3c.png">

"Summary" tab has "Show":
<img width="309" alt="Screen Shot 2020-04-27 at 2 58 18 PM" src="https://user-images.githubusercontent.com/25390369/80420475-0d6ee780-8898-11ea-9c7b-4e583e486851.png">


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
